src/components/UpgradeTableOfContents.tsx

```diff
@@ -145,11 +115,28 @@ const UpgradeTableOfContents: React.FC<IProps> = ({
   }
 
   return (
-    <Aside className={className}>
-      <OuterList>
+    <Box
+      as="aside"
+      className={className}
+      p={0}
+      mb={8}
+      textAlign="end"
+      overflowY="auto"
+      display={{ base: "none", l: "block" }}
+    >
+      <UnorderedList
+        m={0}
+        py={0}
+        ps={4}
+        pe={1}
```

Comment:
Is always using the shorthand our preference?

Reply:
I would say so but I don't have a strong preference.

As a side note, in this particular case, we are going to style the native Chakra component (on the theme definition), as we did with the Button and the Link.